tests/unit/s2n_config_test.c

```diff
@@ -95,5 +96,25 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Test for s2n_config_new() and tls 1.3 behavior */
+    {
+        if (!s2n_is_in_fips_mode()) {
```

Comment:
why is this check required?

Reply:
fips mode returns fips configuration which would be different